Users/coafile.rst

```diff
@@ -39,6 +39,13 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you definitely
+don't want to lint like minified files
+and backup files ( e.g. ``*.min.js``, ``*.orig``).
```

Comment:
Actually you can put here a code-example of a coafile just doing this: ignoring minified and orig-files :D

Reply:
where will I find one?

Reply:
bears = SpaceConsistencyBear
files =
    LICENSE, ISSUE_TEMPLATE, PULL_REQUEST_TEMPLATE.md, MANIFEST.in,
    **.sh, .travis.yml, circle.yml, setup.cfg, ./docs/_static/custom.css
ignore = node_modules/**

Reply:
Is this code ok?